src/Draco.Compiler.Cli/Program.cs

```diff
@@ -18,54 +18,71 @@ internal class Program
 
     private static RootCommand ConfigureCommands()
     {
-        var fileArgument = new Argument<FileInfo>("file", description: "The Draco source file");
+        var fileArgument = new Argument<FileInfo>("source file", description: "The Draco source file");
         var outputOption = new Option<FileInfo>(new string[] { "-o", "--output" }, () => new FileInfo("output"), "Specifies the output file");
         var optionalOutputOption = new Option<FileInfo?>(new string[] { "-o", "--output" }, () => null, "Specifies the (optional) output file");
         var referencesOption = new Option<FileInfo[]>(new string[] { "-r", "--reference" }, Array.Empty<FileInfo>, "Specifies assembly references to use when compiling");
+        var filesOption = new Argument<FileInfo[]>("source files", Array.Empty<FileInfo>, "Specifies draco source files that should be compiled");
```

Comment:
Should we still specify with dash, like `source-files`, in case the user does want to type out the flag?